examples/community/stable_diffusion_reference.py

```diff
@@ -538,7 +538,7 @@ def hack_CrossAttnDownBlock2D_forward(
 
             return hidden_states, output_states
 
-        def hacked_DownBlock2D_forward(self, hidden_states, temb=None):
+        def hacked_DownBlock2D_forward(self, hidden_states, temb=None, **kwargs):
```

Comment:
Why is `kwargs` needed?

Reply:
`hacked_DownBlock2D_forward` is a function they wrote to replace the `forward` method of the `DownBlock2D` ,hence the signature has to match. We added a new argument `scale` for the lora refactor and causes an error here  without `**kwargs`

I think they should write their custom blocks instead